python/graphstorm/config/argument.py

```diff
@@ -3091,8 +3112,16 @@ def _add_link_prediction_args(parser):
             "--gamma",
             type=float,
             default=argparse.SUPPRESS,
-            help="Used in DistMult score func"
+            help="Common hyperparameter symbol gamma."
+    )
+    group.add_argument(
+            "--alpha",
+            type=float,
+            default=argparse.SUPPRESS,
+            help="Common hyperparameter symbol alpha."
     )
+    group.add_argument("--class-loss-func", type=str, default=argparse.SUPPRESS,
+            help="Classification loss function.")
```

Comment:
Can we add choices here like 
```suggestion
    group.add_argument("--class-loss-func", type=str, default=argparse.SUPPRESS,
            help="Classification loss function.", choices=BUILTIN_CLASS_LOSS_FUNCTION)
```

Reply:
We have a check in L2525. 
The launch.py script may not be able to capture the argument error, but it can capture the assert error for sure.